packages/office-addin-dev-settings/src/prompt.ts

```diff
@@ -6,7 +6,7 @@ import { getOfficeAppName, OfficeApp } from "office-addin-manifest";
 
 export async function chooseOfficeApp(apps: OfficeApp[]): Promise<OfficeApp> {
   const questionName = "app";
-  const question: inquirer.Question = {
+  const question: inquirer.ListQuestionOptions<inquirer.Answers> = {
```

Comment:
These inquirer changes were required to get it to build, but I need to manually verify that the prompts still work OK.

Reply:
I validated the prompt using the following command for office-addin-dev-settings:
 `npm run cli -- sideload <manifest>`

Since `--app` is not specified, it prompts for a choice of application.